src/main/java/uk/gov/hmcts/reform/sscs/personalisation/SyaAppealCreatedAndReceivedPersonalisation.java

```diff
@@ -149,7 +149,7 @@ private String buildTextMessageDetails(Subscription subscription, Map<String, St
 
     private Map<String, String> setAppointeeName(Map<String, String> personalisation, SscsCaseData sscsCaseData) {
         Appointee appointee = sscsCaseData.getAppeal().getAppellant().getAppointee();
```

Comment:
minor improvement, first get Appellant object then check for getAppointee() and getIsAppointee()